### DISCUSSION TITLE
Filecoin Loan - Secured Finance

### DISCUSSION THREAD

**Discussion by timowlee** [filecoin-project/community#101] [Project Showcase]
Hi there, 

We are developing a decentralized [Filecoin loan](https://github.com/Secured-Finance/secured-finance-docs/tree/master/basics) platform. Filecoin loan is one of the financial services on [Secured Finance](https://secured-finance.com/). Pls, check out our [Web App](https://secured-finance.on.fleek.co/). Furthermore, we are also developing cross-currency swaps and FX Block Trade and interest rate derivatives, etc. 

**Preview**

![UI-gif](https://user-images.githubusercontent.com/33040281/111365308-a1a19300-8692-11eb-87a5-e85a4ec022f7.gif)

**How to start Filecoin Loan**
- Link your wallets
- Initiate lend and borrow positions
- Confirm your loan/borrow order
- Manage your lending contracts

**Current Status**
- Smart contract: In progress, optimizing order book smart contracts with a new data structure.
- Web App: In progress, developed Subgraph module and integrating with smart contracts.
- Dione oracle network: developed consensus mechanism, integrating Dione node with on-chain dispute mechanism.

**More about Secured Finance**

![Screenshot 2021-03-16 at 20 28 55](https://user-images.githubusercontent.com/33040281/111368486-3e196480-8696-11eb-957e-6ac31d90362e.png)

**Key Features**
- Yield Curve Play
- Loan market
- FX Block Trade
- Cross-currency Swap
- Cross-chain interoperability
- Interests rates swap

**Contact Us**
[Twitter](https://twitter.com/FinanceSecured)
[Telegram](https://t.me/securedfinance)
[Discord](https://discord.gg/JqQKPx8S)

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
hi, can you give all of us an update on this?

**Reply by timowlee** [reply]
Hi f8-ptrk, thank you for asking!
We are still developing the protocol and UI. 
The latest test version is [here](https://app.secured-finance.com/#/)
Currently, we are optimizing the bilateral algorithm which could maximize the usage of collateral. 
Please keep updated with our [Twitter](https://twitter.com/FinanceSecured).
